client/src/app/services/api.service.ts

```diff
@@ -1,28 +1,36 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { HttpClient } from '@angular/common/http';
-import {TaskUpdate, TaskAdd} from '../models/task'
+import { TaskUpdate, TaskAdd } from '../models/task'
+import { UserCredentials } from '../models/user'
```

Comment:
I would just recommend you use such naming convention, where you can define what the variable is by its ending.
for example UserCredentials**Model** or user**Service** or task**Controller**.
It is up to you

Reply:
Changed model names to TaskModel, UserModel etc.